HucaresWF/src/Hucares.Server.Client/MissingPlateClient.cs

```diff
@@ -0,0 +1,155 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Hucares.Server.Client.Models;
+using Newtonsoft.Json;
+
+namespace Hucares.Server.Client
+{
+    public class MissingPlateClient : IMissingPlateClient
+    {
+        private Uri HostUri { get; set; } = new Uri("http://localhost:50510");
+        
+        private readonly HttpClientHelper httpHelper;
+        
+        public MissingPlateClient(HttpClientHelper httpHelper = null)
+        {
+            this.httpHelper = httpHelper ?? new HttpClientHelper();
+        }
+        
+        public async Task<MissingLicensePlate> InsertPlateRecord(string plateNumber, DateTime searchStartDatetime)
+        {
+            var uri = $"api/mlp/insert/{plateNumber}";
+            var fullUri = new Uri(HostUri, uri);
+
+            var plateNumberData = new
+            {
+                plateNumber = plateNumber,
+                searchStartDatetime = searchStartDatetime
+            };
+
+            var jsonContent = JsonConvert.SerializeObject(plateNumberData, new JsonSerializerSettings
+            {
+                NullValueHandling = NullValueHandling.Ignore
+            });
+            
+            var request = new HttpRequestMessage(HttpMethod.Post, fullUri)
+            {
+                Content = new StringContent(jsonContent, Encoding.UTF8, "application/json")
+            };
+
+            return await httpHelper.MakeRequest<MissingLicensePlate>(request);
+        }
+
+        public async Task<IEnumerable<MissingLicensePlate>> GetAllPlateRecords()
+        {
+            var uri = $"api/mlp/all";
+            var fullUri = new Uri(HostUri, uri);
+            var request = new HttpRequestMessage(HttpMethod.Get, fullUri);
```

Comment:
Probably variable instantiation should be separated from the return statement by an empty line